sqlmesh/dbt/context.py

```diff
@@ -85,6 +85,19 @@ def add_variables(self, variables: t.Dict[str, t.Any]) -> None:
         self._variables.update(variables)
         self._jinja_environment = None
 
+    def set_and_render_variables(self, variables: t.Dict[str, t.Any], package: str) -> None:
+        self.variables = variables
+
+        jinja_registry = JinjaMacroRegistry(
+            create_builtins_module="sqlmesh.dbt", top_level_packages=["dbt", package]
```

Comment:
Maybe a constant here?

Reply:
Also, should package precede "dbt" in the top_level_packages order?

Reply:
Why does it matter?

Reply:
If the macro is in both dbt and the top-level-package, which version of the macro is used?

Reply:
actually if we want package macros to take precedence then the existing order is correct.

Reply:
So the last item in the list gets precedence?

Reply:
yes

Reply:
Can we document that somewhere or reverse the list within JinjaMacroRegistry, so that its more intuitive?